### ISSUE TITLE
Latest tag detected incorrectly for double digit patch version

### ISSUE THREAD

**Issue by pbonner-1** [langroodi/tagize#1]
When running the action after creating a "v1.0.10" tag (where v1.0.1 -> v1.0.9 also exist already), the latest tag gets detected incorrectly as v1.0.9. I believe this is due to how the tags are sorted in this line:
- https://github.com/langroodi/tagize/blob/v1.0.0/entrypoint.sh#L80

Using `sort -rV` instead of `sort -r` could solve this:
```bash
  -V, --version-sort          natural sort of (version) numbers within text
```

For the repository I ran into this issue in, sorting without -V results in:
```bash
git tag -l 'v1.*' | cut -d"v" -f2 | sort -r
1.0.9
1.0.8
1.0.7
1.0.6
1.0.5
1.0.4
1.0.3
1.0.2
1.0.10
1.0.1
1.0.0
1.0
```
Whereas sorting with -V results in:
```bash
git tag -l 'v1.*' | cut -d"v" -f2 | sort -rV
1.0.10
1.0.9
1.0.8
1.0.7
1.0.6
1.0.5
1.0.4
1.0.3
1.0.2
1.0.1
1.0.0
1.0
```
Which would result in v1.0.10 being correctly detected as the latest tag.

I imagine a similar problem could also occur for double digit major and minor versions however I have not tested this.

In case more context helps, this is the worklow that was used:
```yaml
name: Update Release Tag Aliases

# Run this workflow for the tag created when a GitHub release is published.
on:
  release:
    types: [published]

jobs:
  release:
    name: Update Release Tag Aliases
    runs-on: [self-hosted, Linux, X64, docker]
    steps:
      - name: Checkout Code
        uses: actions/checkout@v3

      # This will create and/or update semantically versioned tag aliases,
      # e.g. v1 for the latest 1.x.x and v1.2 for the latest 1.2.x etc.
      - name: Update Release Tag Aliases
        uses: langroodi/tagize@d5475d273560a7ee63ea3b41b824b6bb72a44bcd
```

**Comment by langroodi** [comment]
Hi Phillip! Thanks for reporting the issue. You are totally right. The sorting will fail in the mentioned cases.
I will work on that and the fix will be available from **v1.0.1** or **v1** alias.

**Comment by pbonner-1** [comment]
> Hi Phillip! Thanks for reporting the issue. You are totally right. The sorting will fail in the mentioned cases. I will work on that and the fix will be available from **v1.0.1** or **v1** alias.

Thanks for the quick fix! We'll update our workflow to the new version.
